README.md

```diff
@@ -176,7 +176,7 @@ Next, configure the aspectj-maven-plugin to compile-time weave (CTW) the aws-lam
 
 ## Examples
 
-See the **[examples](examples)**  directory for example projects showcasing usage of different utilities.
+See the latest release of the **[examples](https://github.com/aws-powertools/powertools-lambda-java/tree/v1.17.0/examples)** for example projects showcasing usage of different utilities.
```

Comment:
it adds another reference to update after releases...

Reply:
I believe the thing we've got in the release pipeline to replace the code should catch this too 

https://github.com/aws-powertools/powertools-lambda-java/blob/869192a5756f9a0e54d4c840fd2e79a2bda4a0be/.github/workflows/release-prep.yml#L41-L47

🙏  but we will have to watch the next release and confirm.